src/api/interappbus/interappbus.ts

```diff
@@ -61,6 +61,9 @@ export default class InterApplicationBus extends Base {
      * @tutorial InterApplicationBus.send
     */
     public send(destination: Identity, topic: string, message: any): Promise<void> {
+        if (typeof destination !== 'object' || typeof destination.uuid !== 'string') {
+            throw new Error('The destination is not a valid identity object');
```

Comment:
We should reject the promise instead of throwing an exception.

Reply:
Change it.